kazoo/tests/test_watchers.py

```diff
@@ -216,30 +215,18 @@ def changed(children):
             children_watch._run_lock.release()
             time.sleep(retry / 10.0)
 
-        eq_(update.is_set(), False)
-        eq_(children_watch._stopped, True)
-
-    def test_bad_watch_func2(self):
-        counter = 0
-
-        @self.client.DataWatch(self.path)
-        def changed(d, stat):
-            if counter > 0:
-                raise Exception("oops")
-
-        raises(Exception)(changed)
```

Comment:
This test was not doing anything (this expression evaluates to a function object, which is never called).
The test itself looks dubious to me. It tries to raise an exception in the watch, but that will be raised in the thread(let) handling watches, not the main client object context.
There are other tests for the watchers callback. So I think these can be removed.

Reply:
Sounds good to me!